File: 9-palindrome-number/9-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        a=[x]
        sum = 0 
        while x > 0:
            rem = x % 10
            sum = (sum*10) + rem
            x = x//10
        if a[0]==sum:
            return True
        return False

File: 9-palindrome-number/9-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0){
            return false;
        }
        int a=x;
        int temp=0;
        int rem;
        while(a>0){
            rem=a%10;
            temp=(temp*10)+rem;
            a=a/10;  
        }
        if(temp==x){
            return true;
        }
        return false;
    }
}